### DISCUSSION TITLE
Request: Customize Menu maxWidth

### DISCUSSION THREAD

**Discussion by wlumtwilio** [twilio-labs/paste#3586] [General] [answered]
Hi, can the Menu component be updated to allow customization of the maxWidth?

https://paste.twilio.design/components/menu#examples

https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/menu/src/Menu.tsx#L18

**Comment by wlumtwilio** [comment]
Nevermind, I was able to update maxWidth by using inline style

**Reply by SiTaggart** [reply, marked as answer]
Hi @wlumtwilio 

If you're building Twilio products we'd rather you didn't modify any of the styles of Paste components without good, approved by UX reasons.

We lock down our components specifically to stop design drift across our applications to keep a cohesive customer experience.

We'd love to know what particular use case has caused for a need of a different width menu.
